Assets/Placenote/Examples/SpatialCapture/PlacenoteSpatialCaptureSample.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using UnityEngine.UI;
+using UnityEngine.XR.iOS;
+using System.Runtime.InteropServices;
+using System.IO;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+
+
+
```

Comment:
Put this within the SpatialCapture namespace

Reply:
Done.